runchecks.sh

```diff
@@ -16,15 +20,10 @@ echo "Running isort"
 isort --check-only --diff --quiet \
  || { echo "isort errors found! Run 'isort' with no options to fix."; exit 1; }
 
-echo "Running shellcheck"
-git grep -El '^#!/.+\b(bash|sh)\b' | xargs shellcheck
+echo "Running shellcheck (if availablein PATH)"
+if hash shellcheck 2>/dev/null; then
```

Comment:
In some operating system like Mac OS X `shellcheck` is not installed. Skipping it in such cases.

Reply:
Put what?

Reply:
can we put `runchecks.sh` in the `/tests` directory?